tests/layer_tests/tensorflow_tests/test_tf_Round.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestRound(CommonTFLayerTest):
+    def create_tf_round_net(self, input_shape, input_type):
+        tf.compat.v1.reset_default_graph()
+        with tf.compat.v1.Session() as sess:
+            input = tf.compat.v1.placeholder(input_type, input_shape, 'input')
+            tf.raw_ops.Round(x=input)
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+
+    test_data_basic = [
+        dict(input_shape=shape, input_type=type) 
+        for shape in [[6], [2, 5, 3], [10, 5, 1, 5]]
+        for type in [tf.float32, tf.int32, tf.int64, tf.float64]
```

Comment:
please use `@pytest.mark.parametrize` for each parameter. Here you have two parameters: `input_shape` and `input_type`